src/core/integrate.cpp

```diff
@@ -25,6 +25,28 @@
  *  see \ref integrate.hpp "integrate.hpp".
 */
 
+/*
+#########################################
+Fragen:
+-rescale vel/force: warum rescaling?
+-bei calc_forces: warum jedes mal erneut init_forces? 
+-warum inline fkt für inline force???
+-wo werden forces hingeschrieben?
+-warum bond forces bei single particle force?
+-was macht ONEPART_TRACE z.B. bei fene.hpp?
+-was macht die actor list?
```

Comment:
Actors are things that map from a complete particle configuration to forces for all particles.